src/file/metadata.rs

```diff
@@ -354,8 +374,391 @@ impl ColumnChunkMetaData {
       total_uncompressed_size,
       data_page_offset,
       index_page_offset,
-      dictionary_page_offset
+      dictionary_page_offset,
+      statistics
     };
     Ok(result)
   }
 }
+
+/// Column order to compare values with statistics min/max.
+#[derive(Clone, Debug, PartialEq)]
+pub enum ColumnOrder {
```

Comment:
nit: change this to `SortOrder` to be consistent with parquet-mr/cpp?

Reply:
Yes, I will rename it.